provider/lxd/provider.go

```diff
@@ -221,7 +222,9 @@ func (p *environProvider) DetectClouds() ([]cloud.Cloud, error) {
 		configPath := filepath.Join(dir, "config.yml")
 		config, err := p.lxcConfigReader.ReadConfig(configPath)
 		if err != nil {
-			logger.Warningf("unable to read/parse LXC config file (%s): %s", configPath, err)
+			if !strings.HasSuffix(err.Error(), "permission denied") {
```

Comment:
Do we want to ignore the "permission denied" error?

Reply:
yeah cause it only happens when using a strict snap and from what i can tell, the dir under snap/lxd/common is preferred anyway